### DISCUSSION TITLE
Discussion about Sponsorship

### DISCUSSION THREAD

**Discussion by lb-** [dinerojs/dinero.js#723] [General]
I am curious to know whether there is the ability for this project to be sponsored in any way (e.g. via GitHub sponsorship or Open Collective).

It may help some teams be able to support ongoing development of this project outside of only contributions.

I cannot promise a guarantee of sponsorship but just curious if there is a way to do this (I cannot see it on the Readme or on the docs) or a plan to have this option available.

**Comment by sarahdayan** [comment]
Hi, and thanks for asking. Dinero.js is currently not taking donations or sponsorships. I support its development by working at a company, which lets me work on Dinero.js on my free time. Extra money wouldn't necessarily help right now.

I might reconsider later! Moving this to a discussion in the meantime.

**Reply by lb-** [reply]
Great to hear this, thanks for the response.
